src/chat-bubble/interfaces.ts

```diff
@@ -23,8 +23,10 @@ export interface ChatBubbleProps {
   /**
    * Adds aria-label to the chat bubble container. Use this to provide a unique accessible name for each chat bubble on the page.
    * For example, "John Doe at 3:42:10am".
+   *
+   * @deprecated You can safely remove this property as it has no effect.
    */
-  ariaLabel: string;
+  ariaLabel?: string;
```

Comment:
It was possible to use the ARIA label to annotate the messages better, e.g. by adding the timestamp or even just saying "A message from {user}". Besides, the group makes the bubble elements: avatar and message (and can be more in the future) belong together semantically when the bubble is navigated to with a screen-reader. Aren't we missing out on these features? What do we recommend if there is a need to add screen-reader specific annotations to the bubble?

Reply:
We've been talking about options to add it. One option it to make that time information visible in the UI as well. It'd then be available for users with and without assistive technology. This requires more deep dive if we'd consider going that way.

The relevant data now are coming from the avatar and the bubble content itself.

@jorycunningham, do you have any additional thought about?

Reply:
Unless we are showing the timestamp to all users I don't think it is necessary to include it in the accessible name or description of any of the chat bubble elements.

Role="group" with an accessible name from aria-label or -labelledby unfortunately has the effect of hiding the group's content (i.e. the message itself) from aria-live in favor of only reading the accessible name that is applied to the group. 

There is some grouping, and enumeration, of the messages that comes from a given "message unit" (i.e. avatar, message, etc..) by ensuring each "message unit" is contained by a list-item tag in an ordered list.